packages/nvidia_nat_memmachine/tests/test_add_and_retrieve.py

```
@@ -0,0 +1,232 @@
+
+logger = logging.getLogger(__name__)
+
+
+def _memmachine_available(base_url: str) -> bool:
+    """Return True if MemMachine server is reachable."""
+    if not base_url.startswith("http"):
+        base_url = f"http://{base_url}"
+    try:
+        response = requests.get(f"{base_url}/api/v2/health", timeout=5)
+        response.raise_for_status()
+        return True
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)